azext_iot/common/_azure.py

```diff
@@ -46,6 +46,16 @@ def parse_iot_device_module_connection_string(cs):
     return _parse_connection_string(cs, validate, "Module")
 
 
+def parse_iot_endpoint_connection_string(cs):
+    validate = ["Endpoint", "SharedAccessKeyName", "SharedAccessKey"]
```

Comment:
Because this is generically named, can we add a comment that indicates which endpoint types this function supports?

Reply:
Add it as a doc string - under the def, use """ """